### PR TITLE
[BACKLOG-36864] Page Zoom (AA) - Data Source Model Editor Dialog

### PR DESCRIPTION
**Pull request by davidmsantos90**
@pentaho/millenniumfalcon @pentaho/wcag please review

### CODE REVIEW

**Comment by buildguy** [comment]
### :white_check_mark: Build finished in 1m 37s
Build command:
```console
mvn clean verify -B -e -Daudit -amd -pl gwt/impl
```
---
:ok_hand: All tests passed!
 
Tests run: **18**, Failures: **0**, Skipped: **0** &nbsp;&nbsp; [Test Results](https://wingman.orl.eng.hitachivantara.com/job/projects/job/xul/job/PR-249/1/testReport)

 ---
 
 

:information_source:  _This is an automatic message_

File: gwt/impl/src/main/java/org/pentaho/ui/xul/gwt/tags/GwtTabbox.java [hunk 1]
```diff
@@ -63,7 +61,12 @@ public GwtTabbox() {
 
   public GwtTabbox( Orient orient ) {
     super( ELEMENT_NAME );
+
     tabPanel = new TabPanel();
+    tabPanel.setStylePrimaryName( ELEMENT_NAME );
+    tabPanel.addStyleName( VerticalFlexPanel.STYLE_NAME );
+    tabPanel.addStyleName( "with-layout-gap-none" );
```
[the remark this answers is not in the document]
Comment (davidmsantos90) [reply gwt/impl/src/main/java/org/pentaho/ui/xul/gwt/tags/GwtTabbox.java:68]: @dcleao The goal was to remove the gap between the tab and content sections 👍🏼 

I think that creating the classes `gap-none`, `gap-h-none` and `gap-v-none` is a good idea 🚀

**Comment by buildguy** [comment]
### :white_check_mark: Build finished in 2m 2s
Build command:
```console
mvn clean verify -B -e -Daudit -amd -pl gwt/impl
```
---
:ok_hand: All tests passed!
 
Tests run: **18**, Failures: **0**, Skipped: **0** &nbsp;&nbsp; [Test Results](https://wingman.orl.eng.hitachivantara.com/job/projects/job/xul/job/PR-249/2/testReport)

 ---
 
 

:information_source:  _This is an automatic message_

**Review by dcleao — APPROVED** [review]

**Review by mbrasil — APPROVED** [review]

**Comment by buildguy** [comment]
### :white_check_mark: Build finished in 2m 53s
Build command:
```console
mvn clean verify -B -e -Daudit -amd -pl gwt/impl
```
---
:ok_hand: All tests passed!
 
Tests run: **18**, Failures: **0**, Skipped: **0** &nbsp;&nbsp; [Test Results](https://wingman.orl.eng.hitachivantara.com/job/projects/job/xul/job/PR-249/3/testReport)

 ---
 
 

:information_source:  _This is an automatic message_

**Review by dcleao — APPROVED** [review]

**Review by Pancho7 — APPROVED** [review]

**Pull request merged**